lib/factory-helper.rb

```diff
@@ -17,12 +17,21 @@
 module FactoryHelper
   class Config
     @locale = nil
+    @seed = nil
+    @random = Random.new
 
     class << self
       attr_writer :locale
+      attr_reader :random
+
       def locale
         @locale || I18n.locale
       end
+
+      def seed=(seed)
+        @seed = seed
```

Comment:
I'm not sure we need to store `@seed`.  `FactoryHelper::Config.random.seed` should accomplish the same goal while also accessing the seed used if none was explicitly provided.